ui/app/components/balance-component.js

```diff
@@ -0,0 +1,92 @@
+const Component = require('react').Component
+const h = require('react-hyperscript')
+const inherits = require('util').inherits
+
+const { formatBalance, generateBalanceObject } = require('../util')
+
+module.exports = BalanceComponent
+
+inherits(BalanceComponent, Component)
+function BalanceComponent () {
+  Component.call(this)
+}
+
+BalanceComponent.prototype.render = function () {
+  const props = this.props
+  const { balanceValue } = props
+  const needsParse = 'needsParse' in props ? props.needsParse : true
+  const formattedBalance = balanceValue ? formatBalance(balanceValue, 6, needsParse) : '...'
+
+  return h('div.balance-container', {}, [
+    // laptop: 50px 50px
```

Comment:
styling is complete, feel free to make a commit to latest (no need for PR/review process), so unnecessary comments can be removed.

we can also clean it up later, no biggie either way